starlite/kwargs.py

```diff
@@ -252,6 +259,7 @@ def to_kwargs(self, connection: Union[WebSocket, Request]) -> Dict[str, Any]:
         Return a dictionary of kwargs. Async values, i.e. CoRoutines, are not resolved to ensure this function is sync.
         """
         reserved_kwargs: Dict[str, Any] = {}
+        connection_query_params = parse_query_params(connection, self.array_query_parameter_names)
```

Comment:
this shouldn't be here and the parse_query_params interface should not be modified (i.e. it shouldn't receive any other parameters), because it should have an identical output when called as `request.query` and here.